india_stocks_api/brokers/angel.py

```
@@ -165,26 +312,120 @@ def get_history(self, instrument: Equity | Future | Option | Index,
+                        open=float(row[1] or 0),
+                        high=float(row[2] or 0),
+                        low=float(row[3] or 0),
+                        close=float(row[4] or 0),
+                        volume=int(row[5] or 0),
+                        oi=int(row[6]) if len(row) > 6 and row[6] else None,
+                    )
+                    candles.append(candle)
+
+                except Exception:
+                    # skip malformed rows
+                    continue
```

Ruff (0.15.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)